lib/dal/src/component/qualification.rs

```diff
@@ -22,22 +22,10 @@ impl Component {
         ctx: &DalContext,
         component_id: ComponentId,
     ) -> ComponentResult<Vec<QualificationView>> {
-        let component = Self::get_by_id(ctx, component_id).await?;
-
         let mut qualification_views = vec![];
 
-        let qualification_map_value_id = component
-            .attribute_values_for_prop(
-                ctx,
-                RootPropChild::Qualification
-                    .prop_path()
-                    .as_parts()
-                    .as_slice(),
-            )
-            .await?
-            .first()
-            .copied()
-            .ok_or(ComponentError::MissingQualificationsValue(component_id))?;
```

Comment:
No longer need to fetch the component and the function to find the attribute value is now shared